srcpkgs/mattermost-desktop/template

```diff
@@ -1,18 +1,49 @@
 # Template file for 'mattermost-desktop'
 pkgname=mattermost-desktop
 version=4.5.2
-revision=1
-archs="x86_64 i686"
+revision=2
 wrksrc="desktop-${version}"
+archs="x86_64* i686*"
 # electron-builder needs GNU tar
-hostmakedepends="git nodejs-lts-10 python libicns GraphicsMagick xz tar"
-makedepends="libnotify-devel"
+hostmakedepends="git jq nodejs-lts-10 python libicns GraphicsMagick xz tar"
+makedepends="libnotify-devel electron7"
+depends="electron7"
 short_desc="Team messaging app, an open source Slack alternative"
 maintainer="Artem Zhurikhin <ashpool@xecut.net>"
 license="Apache-2.0"
 homepage="https://mattermost.com/"
 distfiles="https://github.com/mattermost/desktop/archive/v${version}.tar.gz"
 checksum=17e32346592d175f091a4f12068e0cfc5cfd20ed7894f88bb5b54b2eb6eac74a
+nocross=yes
```

Comment:
Is it actually nocross, or just because of electron? If it's because of electron, this should be resolved automatically, right?

Reply:
Well, actually it shouldn't matter since the result is technically noarch. So I could remove it again. But there is probably something broken somewhere when cross compiling. I just can't test it now.

Reply:
Guess we need someone to test it.

Reply:
We can't know if cross-compiling doesn't work until we make cross-compile work for electron. I believe that `nocross=yes` is appropriate for now.